one-day-geekweek/agenda.md

```diff
@@ -0,0 +1,71 @@
+class: center, middle
+
+# One Day DevOps / Agile / Cloud Native Geek Week
+
+A single day, heavy duty, partially hands-on introduction to discussions around cloud native applications, the methods used to develop them, the ideas behind the architecture and some of the tools in use.
+
+---
+
+## Prework
+
+* Read [Phoenix Project](http://itrevolution.com/books/phoenix-project-devops-book/)
+* Read [Intro to the Command Line](https://www.gotealeaf.com/books/command_line)
```

Comment:
Disagree here - don't believe that heavy duty CLI understanding is important here.  If you want to find something shorter/smaller I'd reconsider it.